packages/commercetools/composables/src/useLocale/index.ts

```diff
@@ -13,7 +13,8 @@ import {
 } from '@vue-storefront/commercetools-api';
 import { LocaleItem } from '@vue-storefront/commercetools-api/lib/types/setup';
 import Cookies from 'js-cookie';
-import { useSSR, onSSR } from '@vue-storefront/core';
+import { vsfRef } from '@vue-storefront/core';
+import { useAsync } from 'nuxt-composition-api';
```

Comment:
what if someone is not using Nuxt? It has to abstracted the same way `vsfRef` is 
@andrzejewsky isn't `onSSR` already an abstraction ?